commands/FBClassDump.py

```diff
@@ -111,7 +111,7 @@ def run(self, arguments, options):
     struct Block_literal_1 real = *((__bridge struct Block_literal_1 *)$block);
     NSMutableDictionary *dict = (id)[NSMutableDictionary dictionary];
     
-    [dict setObject:(id)[NSNumber numberWithLong:(long)real.invoke] forKey:@"invoke"];
+    [dict setValue:(id)[NSNumber numberWithLong:(long)real.invoke] forKey:@"invoke"];
```

Comment:
That can fix #198 .  There is another MR that can fix it separately

https://github.com/facebook/chisel/pull/241

I guess because function declaration of setValue:forKey doesn't need id <copy>, just id